config/muxcable.py

```diff
@@ -74,7 +74,8 @@ def lookup_statedb_and_update_configdb(per_npu_statedb, config_db, port, state_c
 
     state = get_value_for_key_in_dict(muxcable_statedb_dict, port, "state", "MUX_CABLE_TABLE")
     if state_cfg_val is not configdb_state:
-        config_db.set_entry("MUX_CABLE", port, {"state": state_cfg_val})
+        config_db.set_entry("MUX_CABLE", port, {"state": "active",
```

Comment:
we can fix the lgtm be deleting the `ipv4_value` and `ip64_value`.

BTW, the logic could be simplified further as follows:

```
    if state_cfg_val is not configdb_state:
￼        config_db.set_entry("MUX_CABLE", port, {"state": state_cfg_val})		￼        
     port_status_dict[port] = 'INPROGRESS' if state_cfg_val is 'active' and state is not 'active' else 'OK'		￼        
```

Reply:
also, we are setting the entry to `active` instead of `state_cfg_val`.